packages/api/src/auth/decoders/clerk.ts

```diff
@@ -1,38 +1,28 @@
-import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
+import type IClerk from '@clerk/clerk-sdk-node/instance'
 
-interface Req {
-  event: APIGatewayProxyEvent
-  context: LambdaContext
-}
-
-export const clerk = async (token: string, req: Req) => {
+export const clerk = async (token: string) => {
   // Use require here, to prevent needing clerk sdk in api deps
-  const { sessions, users } = require('@clerk/clerk-sdk-node')
+  const Clerk = require('@clerk/clerk-sdk-node/instance').default
+
+  const { users, base }: IClerk = new Clerk()
```

Comment:
Peter from Clerk team here :) 

By default if the `CLERK_API_KEY` is not set and no `apiKey` parameter is passed on the Clerk instance initialization, it will throw an error as you can see in our [support messages](https://github.com/clerkinc/javascript/blob/main/packages/sdk-node/src/constants/SupportMessages.ts#L2).

Does it fit your case here ? 

If it seems fine the check below is not needed:
```ts
if (!process.env.CLERK_API_KEY) {
    console.error('CLERK_API_KEY env var is not set.')
    throw new Error('CLERK_API_KEY env var is not set.')
  }
```

Reply:
Your messaging looks good. I'll try removing ours and just make sure a helpful message is still surfaced to the user.